src/Core/src/Handlers/View/ViewHandlerOfT.Android.cs

```diff
@@ -2,15 +2,23 @@
 using System.Runtime.CompilerServices;
 using Android.Content;
 using Android.Views;
-using AndroidX.Core.View;
-using Microsoft.Maui;
 using Microsoft.Maui.Graphics;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class ViewHandler<TVirtualView, TNativeView> : INativeViewHandler
 	{
-		View? INativeViewHandler.NativeView => (View?)base.NativeView;
+		View? INativeViewHandler.NativeView => WrappedNativeView;
+		View? INativeViewHandler.ContainerView => ContainerView;
+
+		protected View? WrappedNativeView => ContainerView ?? (View?)NativeView;
+
+		public new WrapperView? ContainerView
```

Comment:
What if, we rename `ContainerView` (currently used by Hot Reload) to `ReloadableView` and keep ContainerView for the Wrapper?